### DISCUSSION TITLE
New Stock Status Option

### DISCUSSION THREAD

**Discussion by arktaur** [inventree/InvenTree#6492] [Ideas]
We have implemented InvenTree as an inventory system for one of our laboratories.

We often deal with devices and instruments getting loaned between departments, but still needing to be tracked.
As such, we were wondering if there was a possibility to add more stock status options to Iventree.

Preferably client side (I could not find options for this in the documentation), but as a Feature Request otherwise.

Specifically, we would like to see an "on loan" or "external" option, that marks the stock as unavailable in the counter.
Additionally, we could forsee the need for an "In use" status with the same effect, when tools or parts are used temporarily by a specific test, experiment or prototype.

Currently the "Quarantined" option is the closest to what we would like to see, but is not conducive to the use of the system by a wider userbase.

Thank you for you considerations.

**Comment by miggland** [comment]
You might want to check this:
https://github.com/inventree/InvenTree/issues/4289

**Reply by matmair** [reply]
I think this will probably address the request. @arktaur please mark @miggland answer if this is what you are looking for. That issue is looking for funding to make it more likely someone will implement it, if your institution can chip in a little we would appreciate it.

**Reply by arktaur** [reply]
Indeed this would solve it. Apologies for not catching that before!
I will close this as resolved.

**Comment by Akshat111111** [comment]
Can I work on this issue ??

**Reply by matmair** [reply]
You are welcome to look at https://github.com/inventree/InvenTree/issues/4289 and https://github.com/inventree/InvenTree/pull/4969; currently, someone is assigned to the issue so I would appreciate you laying out how you would solve the issue in a comment in #4289 so we can gauge if this is something we should/could delegate to you.
We appreciate your interest. This is one of the more involved open tasks so I want to ensure you are not thrown into the cold ocean without being aware of the scope.
